mlos_bench/mlos_bench/schedulers/cyclic_scheduler.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""A simple single-threaded synchronous optimization loop implementation."""
+import logging
+from typing import Any, Dict, List
+from mlos_bench.environments.base_environment import Environment
+from mlos_bench.optimizers.base_optimizer import Optimizer
+from mlos_bench.schedulers.sync_scheduler import SyncScheduler
+from mlos_bench.storage.base_storage import Storage
+from mlos_bench.util import merge_parameters
+
+_LOG = logging.getLogger(__name__)
+
+
+class CyclicScheduler(SyncScheduler):
+    """
+    A simple single-threaded synchronous benchmarking loop implementation
+    that cycles through a list of provided configuration IDs.
+
+    This scheduler does not use any optimizer and is mainly used for benchmarking
+    specific configurations against each other only.
```

Comment:
Let's remove that.

This should be composable with Optimizers.

If you want to only evaluate a small set of tunable-param configs, we have (or can make) other Optimizers (e.g., OneShot) that we can make use of.